app/models/account/states.rb

```diff
@@ -1,5 +1,7 @@
 module Account::States
-  TIME_TO_DELETE_ACCOUNT = 15.days.freeze
+  TIME_TO_DELETE_ACCOUNT        = 15.days.freeze
```

Comment:
`QUIET_PERIOD_BEFORE_DELETION`

Reply:
Done in https://github.com/3scale/porta/pull/267 😄